### DISCUSSION TITLE
Full CLI Update for originals folders

### DISCUSSION THREAD

**Discussion by TyrionWarMage** [photoprism/photoprism#3046] [Q&A]
I'm not using the import feature (because imports are handled by a different software) and therefore i'm using a cron job executing CLI commands for updating Photoprism. However, it seems the "index" command is only adding the images, not doing any processing. Therefore, i'm wondering what is the correct command chain to update everything? Currently i'm using:

1. index
2. faces index
3. faces update
4. places update
5. moments

Is there anything missing or too much? As example, can i restrict some commands to only process new images, not all?

**Comment by lastzero** [comment]
Index does everything that should be needed incl labels and faces. What other processing do you expect?

**Reply by TyrionWarMage** [reply]
> Index does everything that should be needed incl labels and faces. What other processing do you expect?

My test batch did not have any face ids assigned, but then it maybe was only a difficult set. (Halloween party ...) But this means moments and face clusters are also automatically update when running "photoprism index" (incl. cluster assignments for alread existing images)?

**Reply by lastzero** [reply]
Generally yes,. Some tasks may also be performed by the background worker that automatically runs after indexing is complete.
